package.json

```diff
@@ -101,5 +100,6 @@
   "ember-addon": {
     "configPath": "tests/dummy/config",
     "after": "ember-cli-htmlbars"
-  }
+  },
+  "packageManager": "pnpm@9.6.0+sha512.38dc6fba8dba35b39340b9700112c2fe1e12f10b17134715a4aa98ccf7bb035e76fd981cf0bb384dfa98f8d6af5481c2bef2f4266a24bfa20c34eb7147ce0b5e"
```

Comment:
this is added with newer versions of pnpm. 

As-is, there is nothing in the repo saying which pnpm version to use. 

CI says to use version 8 -- so I'll downgrade, set this, and set a volta config.